app/services/solidus_bolt/payments/capture_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Payments
+    class CaptureService < SolidusBolt::BaseService
```

Comment:
Would moving this and related class under `SolidusBolt::Webhooks::Payments::CaptureService` make more sense ?

Since this service seems specific to Webhook requests

Reply:
I don't think that should exist `Sebhook services`. A webhook is just a way to receive a notification, and it should just have to call a service.